oaknational/Oak-Web-Application
# fix: update key config location [LESQ-1202]

## Description

- update config with new key location

## Issue(s)

Fixes #
- invalid key error

## How to test

1. Go to https://deploy-preview-3038--oak-web-application.netlify.thenational.academy and navigate to a lesson download pagewhile feature flagged for login
3. You should be able to login, and there should be no auth related error messages in the console
